nornir_nautobot/plugins/tasks/dispatcher/default.py

```diff
@@ -76,10 +76,11 @@ def get_config(
             logger.log_debug("Substitute lines from configuration based on `substitute_lines` definition")
             running_config = sanitize_config(running_config, substitute_lines)
 
-        make_folder(os.path.dirname(backup_file))
+        if backup_file:
```

Comment:
I presume the point is to send a `None`, I think mypy standards would say `backup_file: Optional[str]`, but would defer to someone else to confirm.

Reply:
`None` will work, but also an empty string would work to keep the type hint the same. I didn't want to move the arg to a kwarg to change the signature.

We already use this pattern with the other arguments of `remove_lines: list, substitute_lines: list` and checking for them to be non-empty with `if remove_lines:` and `if substitute_lines:` so I think this logic is consistent.

Reply:
ok, makes sense.